openfe/setup/atom_mapping/ligandatommapper.py

```diff
@@ -56,5 +56,21 @@ def suggest_mappings(self, componentA: SmallMoleculeComponent,
         # subclasses of this can customize suggest_mappings while always
         # maintaining the consistency that concrete implementations must
         # implement _mappings_generator.
+
         for map_dct in self._mappings_generator(componentA, componentB):
+
+            #TODO: this is a temporary Code snippet - avoids element changes - START
```

Comment:
This should be done at the mapper implementation level, not here

Reply:
Actually I don't know - @richardjgowers ?

Reply:
Was solved during discussions yesterday.